packages/dapp/src/hooks/useBridgeDirection.js

```diff
@@ -24,7 +24,7 @@ export const useBridgeDirection = () => {
 
   const foreignAmbVersion = useAmbVersion(foreignChainId, foreignAmbAddress);
 
-  const homeRequiredSignatures = useRequiredSignatures(
+  const { requiredSignatures, validatorList } = useValidatorsContract(
```

Comment:
Should not be a big difference, but can we request validator contract information from foreign side?
Foreign and home configurations should be identical, but as we execute message on the foreign side, it makes sense to check validator contract there as well.